src/Components/KYRContent/KYRContent.tsx

```diff
@@ -384,23 +384,22 @@ export const GoodCauseExercisingRights: React.FC<
           <li>
             <strong>Tell them it’s unreasonable</strong>
             <br />
-            If your rent increase is more than 10% (or CPI+5%, whichever is
-            lower), tell your landlord it is an unreasonable increase and that a
-            judge could force your landlord to justify it based on increased
-            costs.
+            If your rent increase is more than 8.82%, tell your landlord it is
+            an unreasonable increase and that a judge could force your landlord
+            to justify it based on increased costs.
           </li>
           <li>
             <strong>Withhold the unreasonable increase</strong>
             <br />
             You can withhold the rent increase above the ‘reasonable’ threshold.
-            Pay your old rent plus CPI+5% or 10%, whichever is lower. To be
-            safe, set aside the extra rent in a separate escrow account until
-            your negotiations with your landlord have totally resolved.
+            Pay your old rent plus 8.82%. To be safe, set aside the extra rent
```

Comment:
thoughts on putting 8.82 as a const at the top of this file, under the CPI const? so that in the future when we need to update the copy, it will be slightly easier to just change the constants

Reply:
oh yeah of course, great idea - making that change now